doc/manual/migrating_from_eeglab.rst

```diff
@@ -0,0 +1,68 @@
+Caveats coming from EEGLAB
+==========================
+
+EEGLAB vs. MNE: Pros and Cons
```

Comment:
this section is interesting to read (for me) but feels a bit verbose. I'll let @agramfort comment on the contents.